examples/fuse_examples/imaging/classification/mnist/simple_mnist_starter.py

```diff
@@ -0,0 +1,142 @@
+the goal of this script is simply to train a classifier and show how to wrap a regular pytorch model so it can be
+trained "fuse" style. Meaning it accesses data using keys to a batch_dict. It also demonstrates how easy it is to
+load datasets from fuse.
+"""
+
+import copy
+import os
+from typing import OrderedDict
+
+import torch.nn.functional as F
+import torch.optim as optim
+from torch.utils.data.dataloader import DataLoader
+import pytorch_lightning as pl
+
+from fuse.eval.metrics.classification.metrics_thresholding_common import MetricApplyThresholds
+from fuse.eval.metrics.classification.metrics_classification_common import MetricAccuracy
+
+from fuse.data.utils.samplers import BatchSamplerDefault
+from fuse.data.utils.collates import CollateDefault
+
+from fuse.dl.losses.loss_default import LossDefault
+from fuse.dl.models.model_wrapper import ModelWrapSeqToDict
+from fuse.dl.lightning.pl_module import LightningModuleDefault
+
+from fuseimg.datasets.mnist import MNIST
+from examples.fuse_examples.imaging.classification.mnist import lenet
+
+ROOT = "_examples/mnist"  # TODO: fill path here
+model_dir = os.path.join(ROOT, "model_dir")
+PATHS = {
+    "model_dir": model_dir,
+    "cache_dir": os.path.join(ROOT, "cache_dir"),
+    "inference_dir": os.path.join(model_dir, "infer_dir"),
+    "eval_dir": os.path.join(model_dir, "eval_dir"),
+}
+train_params = {
+    "data.batch_size": 100,
+    "data.train_num_workers": 8,
+    "data.validation_num_workers": 8,
+    "trainer.num_epochs": 12,
+    "trainer.num_devices": 1,
+    "trainer.accelerator": "gpu",
+    "trainer.strategy": "dp",
+    "trainer.ckpt_path": None,
+    "opt.lr": 1e-4,
+    "opt.weight_decay": 0.001,
+}
+
+
+class FuseLitLenet(LightningModuleDefault):
+    def __init__(self):
+        super().__init__()
+        # wrap basic torch model to automatically read inputs from batch_dict and save its outputs to batch_dict
+        self._model = ModelWrapSeqToDict(
+            model=lenet.LeNet(),
+            model_inputs=["data.image"],
+            post_forward_processing_function=lambda logits: (logits, F.softmax(logits, dim=1)),
+            model_outputs=["model.logits.classification", "model.output.classification"],
+        )
+        self._losses = {
+            "cls_loss": LossDefault(
+                pred="model.logits.classification", target="data.label", callable=F.cross_entropy, weight=1.0
+            ),
+        }
+        self._train_metrics = OrderedDict(
+            [
+                ("operation_point", MetricApplyThresholds(pred="model.output.classification")),  # will apply argmax
+                ("accuracy", MetricAccuracy(pred="results:metrics.operation_point.cls_pred", target="data.label")),
+            ]
+        )
+        self._validation_metrics = copy.deepcopy(self._train_metrics)
+        optimizer = optim.Adam(
+            self._model.parameters(), lr=train_params["opt.lr"], weight_decay=train_params["opt.weight_decay"]
```

Comment:
Don't read arguments directly from a global variable. Instead, get it as an argument in __init__